### DISCUSSION TITLE
Is there a way to keep kit open ?

### DISCUSSION THREAD

**Discussion by CoffiDev** [johnlindquist/kit#301] [Q&A]
I would like to have the kit command bar open all the time, and use the shortcut to focus the app and start typing right away, is there a way to configure it? If not, would you recommend I take a look at the app repo https://github.com/johnlindquist/kitapp and try to accomplish it? It is just a "nice to have" that I really want jaja

**Comment by johnlindquist** [comment]
I guess it's technically possible... but here's why I would never do it:

1. The window is a "floating" window that will always be on top of every other window.
2. The window "blur" event is used to cancel the currently running script
3. The window swaps between different UIs for different script shortcuts

In the future Script Kit will support "Script Widgets" which can be permament so people can design their own UIs for interacting with scripts, but that's still pretty far off.

**Reply by CoffiDev** [reply]
yeah, those points make sense to me ... maybe a workaround that could work is to have a shortcut that focuses on a terminal window and automatically run `kit run` so I can directly select the script to run from the command line.

Thanks for the feedback :D and I am excited about the future of kit!
